index.html

```diff
@@ -2,342 +2,377 @@
+                <a href="#">About</a>
+            </li>
+            <li>
+                <a href="#">Contact</a>
+            </li>
+        </ul>
+        <i class="fa-solid fa-bars fa-xl"></i>
+    </nav>
+
+  </header>
+  <div id="Header-background">
+      <div class="ellipse-rectangle-6"></div>
+      <div class="ellipse-big-half-cercle-top"></div>
+      <div class="ellipse-Rectangle-7"></div>
+      <div class="ellipse-big-cercle-standby"></div>
+      <div class="ellipse-Rectangle-4"></div>
+      <div class="ellipse-big-half-donut-active"></div>
+
+      <section class="headline">
+
+          <div class="intro-container">
+            <div class="headline-intro">
+                <h1>I’m Massaillon
+                    Glad to see you!</h1>
             </div>
-            <div class="see-project">
-                <div><a class="button-project" href="#">See Project</a></div>
-            </div>
-
-        </div>
-
-
-        <div class="portfolio-items">
-
-            <div class="slider-work-4"></div>
-            <h3 class="title-work-1">Multi-Post
-                Stories</h3>
-            <div class="detail-overview-work-1">
-                <div class="">CANOPY</div>
-                <div class="counter"></div>
-                <div>Back End Dev</div>
-                <div class="counter"></div>
-                <div>2015</div>
-            </div>
-            <div class="desc-text-work-1">
-                <p>A daily selection of privately personalized
-                    reads;
-                    no accounts or sign-ups required.</p>
-            </div>
-            <div class="button-work-1">
-                <ul>
-                    <li>
-                        <a class="button-html" href="#">html</a>
-                    </li>
-                    <li>
-                        <a class="button-css" href="#">css</a>
-                    </li>
-                    <li>
-                        <a class="button-js" href="#">javascript</a>
-                    </li>
-                </ul>
-            </div>
-            <div class="see-project">
-                <div><a class="button-project" href="#">See Project</a></div>
-            </div>
-
-        </div>
-    </main>
-
-    <section>
-        <div class="container-about">
-            <div class="About-Myself">
-                <h2>About
-                    Myself</h2>
-            </div>
-            <div class="intro-desc-sumary">
-                <p>Hello I’m a software developer!
-                    I can help you build a product
-                    , feature or website Look through some
-                    of my work and experience!
-                    If you like what you see and have a project
-                    you need coded, don’t hestiate to contact me.</p>
-            </div>
-
-
-            <div class="social-area">
+              <div class="intro-desc-sumary">
+                  <p>I’m a software developer! I can help you build a product ,
```

Comment:
Kindly make sure that your about section content is centered with the appropriate `left` and `right` margins according to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1):

Your Design:
![image](https://user-images.githubusercontent.com/38088818/165941293-7f6d2541-a89e-4736-89d2-93ddf46ce5db.png)


Figma:
![image](https://user-images.githubusercontent.com/38088818/165940957-f2ee919a-d094-4a4a-9d4e-9108a02cc98b.png)